src/ShardRouting.php

```diff
@@ -0,0 +1,20 @@
+<?php declare(strict_types=1);
+
+namespace ElasticScoutDriverPlus;
+
+/**
+ * Use this trait to enable custom elasticsearch routing functionality.
+ * https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-routing-field.html
+ */
+trait ShardRouting
+{
+    /**
+     * Override this method to specify a document field that will be used
+     * to route the document to a shard.
+     * By default the scout key will be used.
+     */
+    public function getRoutingPath(): string
```

Comment:
Maybe we should define this method as an abstract? I doubt, there will be a use case when someone needs to route documents by scout key. By defining it as an abstract, we force devs to define the value.